src/ListenerService.ts

```diff
@@ -118,12 +94,12 @@ export class ListenerService implements ConnectionHeartbeatListener {
             return deferred.promise;
         }
         const registrationKey = this.userRegistrationKeyInformation.get(userRegistrationKey);
-        const registerRequest = registrationKey.getRegisterRequest();
+        const registerRequest = registrationKey.getRegisterRequest().copyWithNewCorrelationId(-1);
```

Comment:
nit: add `New correlation id will be set on the invoke call` comment like it's done in `registerListener()`